forward_engineering/dbtProvider.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @typedef {import('./types').ColumnDefinition} ColumnDefinition
+ */
+
+module.exports = ({ app }) => {
+	const { decorateType } = require('./helpers/columnDefinitionHelper')(app);
+
+	return {
```

Comment:
this should probably be a class with a factory function taking  the app instance as a param. Especially because we have the build system in place